### PR TITLE
Italian localization

### PR DESCRIPTION
**Pull request by claudiopro**
Hi @asgeirn @stephenc @kohsuke I'd love to contribute an Italian translation for this plugin :smile: let me know if you need a reviewer for proofreading, I could easily find one.

### CODE REVIEW

File: src/main/webapp/index_it.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Strict//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Palle Verdi</title>
```
Comment (nfalco79) [inline src/main/webapp/index_it.html]: What do you think about "Icone" instead of "palle"?

File: src/main/webapp/index_it.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+</head>
+<body>
+<h1>Palle Verdi</h1>
+<p>Il plugin Palle Verdi &egrave; stato installato con successo. Se ha funzionato correttamente, dovresti vedere soltanto icone verdi qui di seguito:</p>
```
Comment (nfalco79) [inline src/main/webapp/index_it.html]: Personally "Se ha funzionato correttamente, di seguito dovresti vedere soltanto icone verdi:" sounds better.

File: src/main/webapp/index_it.html (hunk 1 above)
Comment (claudiopro) [inline src/main/webapp/index_it.html]: Do you think the form "palle" is too derogatory in this context? Blue balls itself _is_ derogatory.

File: src/main/webapp/index_it.html (hunk 2 above)
Comment (claudiopro) [inline src/main/webapp/index_it.html]: You're right, I am always unsure whether to modify the sentence or stick to a word-by-word translation, and tend to be too conservative with the latter.

File: src/main/webapp/index_it.html (hunk 1 above)
Comment (nfalco79) [inline src/main/webapp/index_it.html]: A little bit. Also the french translation use icon instead ball.
Mine was only a tickle to detect a possible variant :)

**Comment by nfalco79** [comment]
:+1: for merge

**Comment by claudiopro** [comment]
@nfalco79 I'll fix the wording as suggested. Do you want me to squash commits?

**Pull request merged**